Assets/Scripts/Models/Character/Character.cs

```diff
@@ -757,31 +736,33 @@ private void Update_DoMovement(float deltaTime)
         if (nextTile == null || nextTile == CurrTile)
         {
             // Get the next tile from the pathfinder.
-            if (pathAStar == null || pathAStar.Length() == 0)
+            if (movementPath == null || movementPath.Count == 0)
             {
                 // Generate a path to our destination.
                 // This will calculate a path from current tile to destination tile.
-                pathAStar = new Path_AStar(World.Current, CurrTile, DestTile);
-                if (pathAStar.Length() == 0)
+                movementPath = Pathfinder.FindPathToTile(CurrTile, DestTile, MyJob != null ? MyJob.adjacent : false);
+                if (movementPath == null || movementPath.Count == 0)
                 {
                     Debug.ULogErrorChannel("Character", "Path_AStar returned no path to destination!");
```

Comment:
If I leave the game running for a few minutes game time (about a minute at 4x) I start getting spammed by this every tick.